pages/studentPortal/accountability/Dropdown.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+
+function Dropdown({ label, options, onSelect }) {
```

Comment:
This is a generic and reusable component. Put it in the ui folder where the button lives so everyone uses this dropdown component in the future

Reply:
Sounds Good!